en/setup/dev_env_linux_centos.md

```diff
@@ -27,72 +19,32 @@ easy_install toml
 yum install openocd libftdi-devel libftdi-python python-argparse flex bison-devel ncurses-devel ncurses-libs autoconf texinfo libtool zlib-devel cmake
 ```
 
-Note:You may want to also install  python-pip and screen
+> **Note** You may want to also install  python-pip and screen
+
+## GCC Toolchain Installation
+<!-- import GCC toolchain common documentation -->
+{% include "_gcc_toolchain_installation.txt" %}
 
-#### Additional 32 bit libraries
+## Additional 32 bit libraries
 
 Once the arm toolchain is installed test it with:
 
-<div class="host-code"></div>
-
 ```sh
 arm-none-eabi-gcc --version
 ```
 If you receive the following message
 
-<div class="host-code"></div>
-
 ```sh
 bash: gcc-arm-none-eabi-4_7-2014q2/bin/arm-none-eabi-gcc: /lib/ld-linux.so.2: bad ELF interpreter: No such file or directory
 ```
 Then you will also need to install other 32-bit libraries glibc.i686 ncurses-libs.i686
 
-<div class="host-code"></div>
-
 ```sh
 sudo yum install glibc.i686 ncurses-libs.i686
 ```
 
 > **Note** Pulling in ncurses-libs.i686 will pull in most of the other required 32 bit libraries. Centos 7 will install most all the PX4 related devices without the need for any added udev rules. The devices will be accessible to the predefined group ' dialout'. Therefore any references to adding udev rules can be ignored. The only requirement is that your user account is a member of the group 'dial out'
 
 
-### Arch Linux
-
-<div class="host-code"></div>
-
-Ensure you have the multilib repository enabled.
-
-```sh
-sudo pacman -S base-devel lib32-glibc git-core python-pyserial python-numpy python-pip zip vim
-pip install --user toml
-```
-
-Install [yaourt](https://archlinux.fr/yaourt-en) (Yet AnOther User Repository Tool), a package manager for the [Arch User Repository (AUR)](https://wiki.archlinux.org/index.php/Arch_User_Repository).
-
-Then use it to download, compile and install the following:
-
-<div class="host-code"></div>
-
-```sh
-yaourt -S genromfs python-empy
-```
-
-#### Permissions
-
-The user needs to be added to the group "uucp":
-
-<div class="host-code"></div>
-
-```sh
-sudo usermod -a -G uucp $USER
-```
-
-After that, logging out and logging back in is needed.
-
-
-> **Note** Log out and log in for changes to take effect! Also remove the device and plug it back in!**
-
-
-### GCC Toolchain Installation
-<!-- import GCC toolchain common documentation -->
-{% include "_gcc_toolchain_installation.txt" %}
+<!-- import docs ninja build system -->
```

Comment:
@jlecoeur @dagar According to Lorenz, ninja is recommended for all platforms. The import here downloads a zip and installs in home directory. Is there a version in package manager that can be used instead? Shouldn't we have this in the scripts?

Reply:
Yes
sudo pacman -S ninja

Reply:
Can you add to the script?

Reply:
Done

Reply:
Thanks. I'll remove my "manual" installation instructions :-)